test/integration_test.py

```diff
@@ -769,14 +772,24 @@ def _prepare_notifications(self, catalog: CatalogName) -> Tuple[JSONs,
                                                                     Set[SourcedBundleFQID]]:
         reindex_notifications = []
         fqids = set()
-        for source, prefix, new_fqids in self._list_bundles(catalog):
+
+        def update(source: SourceRef, prefix: str, new_fqids: Set[SourcedBundleFQID]):
             reindex_notifications.append({
                 'action': 'reindex',
                 'catalog': catalog,
                 'source': source.to_json(),
                 'prefix': prefix
             })
             fqids.update(new_fqids)
+
+        for source_bundle_info in self._list_managed_access_bundles(catalog):
+            update(*source_bundle_info)
```

Comment:
the use of `*source_bundle_info` hides a type warning that the returned tuple is a `(SourceRef, str, List)` per the type hint of the `_list_*` methods. while the `update` method expects the `fqid` arg to be a `set`

<img width="1079" alt="Screen Shot 2021-11-04 at 1 07 10 PM" src="https://user-images.githubusercontent.com/1947534/140412177-139e347d-a7e1-42aa-a66b-1e89b5b12ef5.png">

Reply:
The type annotation was too strict anyway, I incorrectly assumed `set.update` required a `set` but it just needs an `Iterable`. Nice catch.